manim/animation/numbers.py

```diff
@@ -16,6 +16,41 @@
 
 
 class ChangingDecimal(Animation):
+    """Animate a :class:`~.DecimalNumber` to values specified by a user-supplied function.
+
+    Parameters
+    ----------
+    decimal_mob
+        The :class:`~.DecimalNumber` instance to animate.
+    number_update_func
+        A function that returns the number to display at each point in the animation.
+    suspend_mobject_updating
+        If ``True``, the mobject is not updated outside this animation.
+
+    Raises
+    ------
+    TypeError
+        If *decimal_mob* is not an instance of :class:`~.DecimalNumber`.
```

Comment:
Could you please use backticks instead for this one?
```suggestion
        If ``decimal_mob`` is not an instance of :class:`~.DecimalNumber`.
```